build.sh

```diff
@@ -10,3 +10,9 @@ for path in test/Serilog.Tests/project.json; do
     dotnet build ${dirname} -f netcoreapp1.0 -c Release
     dotnet test ${dirname} -f netcoreapp1.0  -c Release
 done
+
+for path in test/Serilog.PerformanceTests/project.json; do
+    dirname="$(dirname "${path}")"
+    dotnet build ${dirname} -f netcoreapp1.0 -c Release
+    dotnet test ${dirname} -f netcoreapp1.0  -c Release
```

Comment:
Looks like this will cause the perf tests to run on Travis. Is that the intention?